### PR TITLE
Ethereals now discharge electricity if their charge is too high

### PR DESCRIPTION
**Pull request by wesoda25**
## About The Pull Request

Ethereal cap energy has been increased from 100 to 150. 75-100 is still ideal power range, from 100-125 you will begin to take minor toxin damage. Above 125, you take more toxin damage, and there is a 10% chance each tick for you to begin the discharge process. Which I will include a video of below. There is a 10% to get a heart attack during it to dissuade "tactical discharging", if you can call it that. Shock is weak. Idea oked by @Qustinnus 

fH0o5bZG8m

## Why It's Good For The Game

Adds more interesting mechanics to Ethereals, who peoples primary complaint is that they are bland. Ethereals need to regulate their charge better now. 

## Changelog
:cl:
add: Ethereal max charge has been increased from 100 to 150. If your charge is between 100-125 you will take minor toxin damage. If above 125, you have a chance to violently discharge excess energy, which stabilize your energy level while causing some cool effects. 
imageadd: Ethereal overcharged and supercharged HUD Alert sprites
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
Is my cl detailed enough?

Also, just to make it clear: there is no combat advantage to ethereals forcing themselves to discharge. They take damage from it, are stunned longer than anyone else around them, have a high brutemod when their charge is high (once I push something waiting on my local), and have a chance of developing heart disease. If anything I'm more worried that people might take advantage of ethereal discharging to kill them, but tbh they could just use toxins.

**Comment by Iatots** [comment]
Any easy way to make the flash a bit or vibrate?

**Comment by itseasytosee** [comment]
Great now all we need is lore and it might be able to compete with those perfectly coded plasmemes

**Comment by wesoda25** [comment]
> Any easy way to make the flash a bit or vibrate?

idk what you mean by this. if by flash you mean the white screen overlay, I could yeah but idk if I want to. I'll see for screen shake too since I think thats what you mean, I'm just using whatever looks the best.

**Comment by AdamElTablawy** [comment]
it won't be for combat but it'll definitely be griefy

could also be used as a shitty suicide tactic

maybe no stun on the other party?

**Comment by mrhugo13** [comment]
On one hand i think this is kinda cool and adds some nice functionality to Ethereals.

On the other hand i am incredibly concerned that this will turn Ethereals into a race that's less just being somewhat interesting and more just being a race nobody likes because they are so annoying to deal with.

Additionally, i am worried that this would be a griff tactic.

Also quick question, who is complaining that Ethereals are bland? I haven't exactly heard anything that come close to those complaints currently.

**Comment by Iatots** [comment]
> 
> 
> > Any easy way to make the flash a bit or vibrate?
> 
> idk what you mean by this. if by flash you mean the white screen overlay, I could yeah but idk if I want to. I'll see for screen shake too since I think thats what you mean, I'm just using whatever looks the best.

I meant making their bioluminescence have a strobe effect, and their mob sprite vibrate like when people get full of stimulants.

**Comment by necromanceranne** [comment]
With enough tesla coils I wonder if I can become a living engine.

**Comment by Qustinnus** [comment]
I guess I can write up ethereal lore

**Comment by Qustinnus** [comment]
@mrhugo13 its the main complaint I get about ethereals whenever i aks people. Its a combi of 'no mechanics" and "no cool visuals besides light/color"

File: code/datums/mood_events/needs_events.dm [hunk 1]
```diff
@@ -20,6 +20,14 @@
 	mood_change = -10
 
 //charge
+/datum/mood_event/supercharged
+	description = "<span class='boldwarning'>I can't possibly keep all this power inside, I need to release some quick!</span>\n"
```
Comment (Qustinnus) [inline code/datums/mood_events/needs_events.dm]: ethereal version of i need to COOOOOOOOM

Comment (wesoda25) [reply code/datums/mood_events/needs_events.dm]: 😏

**Comment by wesoda25** [comment]
Away from computer but once I get back I _could_ remove either the stun or damage tag from the tesla zap, which would help with grief. But I think honestly the problem would resolve itself since ethereals have no asimov protection, and thus an overloading ethereal is a health hazard to humans. If it becomes an issue I could always make a pr making the tesla cosmetic for everyone but the ethereal.

**Comment by Ohagi-Chan** [comment]
Does toxin damage make the most sense for overcharging, or could organ/cell damage be considered?

**Comment by wesoda25** [comment]
After thinking about it the concerns of this being used solely to grief are valid, so I’m going to make it so that the tesla arcs neither stun nor do damage, they will basically be purely cosmetic. I’ll commit the changes later today

**Comment by wesoda25** [comment]
DOne

**Comment by TheSilverNuke** [comment]
Sad. It should give a small amount of burn damage, and make them a little jittery/dizzy/confused IMO. It would be a valid self defense option but not so powerful that it makes it worth griefing with.

**Comment by wesoda25** [comment]
> Sad. It should give a small amount of burn damage, and make them a little jittery/dizzy/confused IMO. It would be a valid self defense option but not so powerful that it makes it worth griefing with.

Its not supposed to be a self defense option

**Comment by ExcessiveUseOfCobblestone** [comment]
Which maintainer okayed this?

**Comment by wesoda25** [comment]
Qustinnus

**Comment by ExcessiveUseOfCobblestone** [comment]
he's not a maintainer 😳 

image

**Comment by TheSilverNuke** [comment]
Why was merging blocked?

**Comment by wesoda25** [comment]
bruh why does it say member if hes not a maintainer FUCK

ok umm this is awkward I am now open to maintainer sponsorship's, mr ShizCalev I notice you upvoted this, would you like to endorse this PR as well as my other ethereal one?

**Comment by Qustinnus** [comment]
lmao

**Comment by ExcessiveUseOfCobblestone** [comment]
> 
> 
> Why was merging blocked?

we have a rule regarding species changes that requires maintainer approval.

@wesoda25 lol'd

**Comment by wesoda25** [comment]
uhh so was this approved?

**Comment by Jordie0608** [comment]
¯\_(ツ)_/¯

**Pull request merged**